spec/ic.did

```diff
@@ -116,8 +116,8 @@ service ic : {
     headers: vec http_header;
     body : opt blob;
     transform : opt record {
-      function : func (record {response : http_response; context : blob}) -> (http_response) query;
-      context : blob
+      function : func (record {response : http_response; arg : blob}) -> (http_response) query;
+      arg : blob
```

Comment:
Why this change?

Reply:
Good question :) I don't actually know why it showed up. Maybe it is some old change that slipped into my branch.